src/views/Alteration.vue

```diff
@@ -210,6 +222,12 @@ export default class Alteration extends Mixins(
   @Prop({ default: false })
   readonly appReady: boolean
 
+  /** Whether to show the Transactional Folio Number section. */
+  private get showTransactionalFolioNumber (): boolean {
+    // TODO: implement
+    return false // (this.isPremiumAccount && !this.isRoleStaff)
+  }
```

Comment:
When will this be shown in the future? 
Sorry, i'm not familiar with the `Transactional FN` 😁

Reply:
Sounds good, thanks!